db/migrate/20210224154303_add_requested_scopes_to_zaikio_access_tokens.rb

```diff
@@ -0,0 +1,6 @@
+class AddRequestedScopesToZaikioAccessTokens < ActiveRecord::Migration[6.1]
+  def change
+    add_column :zaikio_access_tokens, :requested_scopes, :string, array: true, default: [], null: false
+    Zaikio::AccessToken.update_all("requested_scopes = scopes")
```

Comment:
We should also touch the records `updated_at` somehow, e.g.:

```suggestion
    Zaikio::AccessToken.update_all("requested_scopes = scopes, updated_at = now()")
```

Reply:
good point